### PR TITLE
added num of reviews

### PR DESCRIPTION
**Pull request by ayush-shah12**
added number of critic reviews

### CODE REVIEW

File: tests/test_movie.py
Comment (preritdas) [inline tests/test_movie.py:1]: Seems like this test would fail as soon as another review is added?

File: tests/test_standalone.py [hunk 1]
```diff
@@ -72,3 +72,8 @@ def test_critics_consensus(happy_gilmore_content):
     assert standalone.critics_consensus("happy gilmore", happy_gilmore_content) == \
         "Those who enjoy Adam Sandler's schtick will find plenty to love in this gleefully juvenile take on professional golf; "+\
         "those who don't, however, will find it unfunny and forgettable."
+
+def test_num_of_reviews(happy_gilmore_content):
+    assert standalone.num_of_reviews("happy gilmore", happy_gilmore_content) == 55
```
Comment (preritdas) [inline tests/test_standalone.py:77]: Same thing here... seems like this test will fail as soon as another review is added? 

Perhaps check that the value is (1) a valid integer _and greater_ than 55?

**Review by preritdas — CHANGES_REQUESTED** [review]
Overall great just some questions about the tests... Testing number of reviews with a hardcoded current value doesn't seem like the best approach, because as soon as someone writes a new review the test will fail (correct me if I'm wrong). 

Would suggest testing for:
1. Is a valid `int`
2. Is greater than or equal to the current value

File: tests/test_movie.py
Comment (ayush-shah12) [reply tests/test_movie.py:1]: good point, updated tests!

**Review by preritdas — APPROVED** [review]

**Pull request merged**